lib/test/drm/cenc/getPSSHData.test.ts

```diff
@@ -1,7 +1,7 @@
-import { getPSSHData } from '@svta/common-media-library/drm/cenc/getPSSHData';
+import { getPSSHData } from '@svta/common-media-library/drm';
 import { strictEqual } from 'node:assert';
 import { describe, it } from 'node:test';
-import { samplePsshBox } from '../common/samplePsshBox';
+import { samplePsshBox } from '../common/samplePsshBox.ts';
```

Comment:
I thought that we have to postfix imports with .js, not .ts. Am I missing something?

Reply:
The tests are run with Node's type-stripping feature, so in the test files you need to import the files with their proper file extension. File imported from the `test` directory will have `.ts`, and files imported from the compiled library with have `.js`.